src/components/todolistcrud.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable max-classes-per-file */
+import renderTodo from './render.js';
+import TodoStore from './localstorage.js';
+
+class TodoList {
+  constructor(description, isComplete = false) {
+    this.id = Date.now().toString(36) + Math.random().toString(36).substr(2);
```

Comment:
- [ ] **Required:** Id should a number. It should be updatable.

![image](https://user-images.githubusercontent.com/27709832/161283458-01bc35dd-6d49-4ede-a8e5-bb32d6d70714.png)